src/app.controller.ts

```diff
@@ -11,15 +11,18 @@ export class AppController {
   constructor(private readonly appService: MonitorService) { }
 
   /**
-   * Get requests that attempt to fetch the endpoints will be handled here
+   * Get requests that attempts to fetch all endpoints will be handled here
```

Comment:
hier geht es um mehrere requests, daher kein singular s
```suggestion
   * Get requests that attempt to fetch all endpoints will be handled here
```